content/backend/Directory.Build.targets

```diff
@@ -60,42 +62,43 @@
     <PackageReference Update="Microsoft.Extensions.Http" Version="$(ExtensionsVersion)" />
     <PackageReference Update="Microsoft.Extensions.Http.Polly" Version="$(ExtensionsVersion)" />
 
+    <PackageReference Update="Microsoft.Extensions.Azure" Version="1.2.0" />
+
     <PackageReference Update="Microsoft.AspNetCore.Identity.EntityFrameworkCore" Version="$(AspNetCoreVersion)" />
     <PackageReference Update="Microsoft.AspNetCore.Authentication.JwtBearer" Version="$(AspNetCoreVersion)" />
     <PackageReference Update="Microsoft.AspNetCore.Mvc.Testing" Version="$(AspNetCoreVersion)" />
+    <PackageReference Update="Azure.Extensions.AspNetCore.DataProtection.Blobs" Version="1.2.1" />
+    <PackageReference Update="Azure.Extensions.AspNetCore.DataProtection.Keys" Version="1.1.0" />
 
     <PackageReference Update="MassTransit" Version="$(MassTransitVersion)" />
     <PackageReference Update="MassTransit.Azure.ServiceBus.Core" Version="$(MassTransitVersion)" />
     <PackageReference Update="MassTransit.RabbitMQ" Version="$(MassTransitVersion)" />
 
     <PackageReference Update="OpenTelemetry" Version="1.2.0-beta1" />
-    <PackageReference Update="OpenTelemetry.Contrib.Instrumentation.MassTransit" Version="1.0.0-beta2" />
-    <PackageReference Update="OpenTelemetry.Exporter.OpenTelemetryProtocol" Version="1.2.0-beta1" />
-    <PackageReference Update="OpenTelemetry.Exporter.Zipkin" Version="1.2.0-beta1" />
-    <PackageReference Update="OpenTelemetry.Extensions.Hosting" Version="1.0.0-rc8" />
-    <PackageReference Update="OpenTelemetry.Instrumentation.AspNetCore" Version="1.0.0-rc8" />
-    <PackageReference Update="OpenTelemetry.Instrumentation.Http" Version="1.0.0-rc8" />
-    <PackageReference Update="OpenTelemetry.Instrumentation.SqlClient" Version="1.0.0-rc8" />
+    <PackageReference Update="OpenTelemetry.Exporter.OpenTelemetryProtocol" Version="1.3.0" />
```

Comment:
This requires `OpenTelemetry` to be `1.3.0` or later, so you might as well also update the line above :slightly_smiling_face: